src/main/java/org/folio/marc4ld/configuration/property/Marc4BibframeRules.java

```diff
@@ -69,6 +77,7 @@ public void putMappings(Map<String, Character> mappings) {
   @Data
   public static class Marc2ldCondition {
     private Map<Character, String> fields;
+    private Map<Character, String> fieldsOneOf;
```

Comment:
I think it may be more readable if we rename the existing `fields` to `fieldsAllOf`
```
private Map<Character, String> fieldsAllOf;
private Map<Character, String> fieldsOneOf;
```

Reply:
Create, change it